src/index.html

```diff
@@ -6,9 +6,59 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home">
+        <img src="./images/logo.png"
+          alt="moyo"
+          class="img-logo"
+        >
+      </a>
+
+      <nav class="nav-list">
+        <ul class="is-active nav__list">
```

Comment:
ul - is a list, on the list should be items. Now you have many lists, but you need to create one list with many items
```
<ul>
  <li></li>
	<li></li>
	<li></li>
</ul>
```